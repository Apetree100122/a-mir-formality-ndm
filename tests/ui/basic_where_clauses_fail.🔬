[
    crate core {
        trait A<ty T> where [T: B<>] { }

        trait B<> where [] { }

        trait WellFormed<> where [for<ty T> u32: A<T>] { }
    }
]
