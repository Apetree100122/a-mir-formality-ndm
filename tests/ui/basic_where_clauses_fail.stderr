Error: check_trait(WellFormed)

Caused by:
    0: prove_where_clause_well_formed(for <ty> u32 : A < ^ty0_0 >)
    1: prove_where_clause_well_formed(u32 : A < !ty_2 >)
    2: failed to prove {@ WellFormedTraitRef(A(u32, !ty_2))} given {for <ty> A(u32, ^ty0_0)}, got {}
